bindata/assets/alerts/kube-apiserver-slos-extended.yaml

```diff
@@ -0,0 +1,189 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
so this is just the original file copied into a new name, right?

Reply:
The original file contents were distributed to two separate files, one of which (`extended`) is skipped on SNO installs.